src/components/Terminal.jsx

```diff
@@ -120,11 +118,12 @@ export default class Terminal extends React.Component {
   }
 
   showHelp () {
-    for (let c in this.state.commands) {
-      const cmdObj = this.state.commands[c]
-      const usage = cmdObj.usage ? ` - ${cmdObj.usage}` : ''
-      this.pushToStdout(`${c} - ${cmdObj.description}${usage}`)
-    }
+    const helpLabelFn = this.props.helpLabel
```

Comment:
When running this out of the box, I just get completely garbled output. The command I entered is also not displayed.

![Garbled output](https://i.linuswillner.me/RRkQsm2.png)

Reply:
Sorry but this is a bug if you use tag 'dangerMode={true}', a react element would insert into a __html, rather than render directly.